distribution/docs/src/main/resources/content/config.adoc

```diff
@@ -17,7 +16,9 @@ Version ${project.version}. Copyright (c) Codice Foundation
 :release-notes-url: https://codice.atlassian.net/wiki/spaces/DDF/pages/71275152/Release+Notes
 :adoc-include: ${project.build.directory}/doc-contents/_contents
 :hardening-step: Required Step for Security Hardening
-:title-logo: logo_ddf.png
+:title-logo-image: image::logo_ddf.png[]
+//:title-page-background-image: image::background.png[]
```

Comment:
❔ Did you mean to keep the `//` here?

Reply:
We really don't have anything to use as a background image, so I'll take it out